web/cobrands/tfl/base.scss

```diff
@@ -183,6 +188,18 @@ ol.big-numbers>li {
     }
 }
 
+#front-recently {
```

Comment:
I don't think this one is needed, the `.item-list--reports li > a` below should also catch it.

Reply:
Thanks @dracos, fixed